core/sql/regress/core/TEST002

```diff
@@ -613,8 +623,44 @@ select
 (select count(*) from t002sub limit 20) as result_value
 from t002main;
 
--- Should return 7
+-- Should return 2
 select x from (select x from t002sol order by x desc limit 1);
+
+-- Tests of [first/any/last n] and limit n with ORDER BY in subqueries
+-- (Note that limit n is the same as [any n])
+
+prepare xx from select val from t002x1 where val in (select [first 1] val from t002x2 order by val);
+
+-- query plan should be a sequence plan, not a firstn plan; the ORDER BY causes a rewrite
+explain options 'f' xx;
+
+-- should return one row, -1865644273
+execute xx;
+
+prepare xx from select val from t1 where val in (select [any 1] val from t2 order by val);
```

Comment:
The table name here should be t002x1, not t1. (Found this when I reran this test on a workstation after cleaning my instance.) Similarly for t2.